### PR TITLE
Modified deployment config for workshop

### PR DESCRIPTION
**Pull request by suppathak**

### CODE REVIEW

File: notebooks/time-to-merge-prediction/workshop/deployment-config.yaml [hunk 1]
```diff
@@ -26,28 +26,25 @@ spec:
...
-                key: S3_BUCKET
-                name: thoth-github-pr-ttm
-          - name: S3_MODEL_KEY
-            valueFrom:
-              secretKeyRef:
-                key: S3_MODEL_KEY
-                name: thoth-github-pr-ttm
+                key: AIOPS_TOOLS_WORKSHOP_BUCKET
+                name: s3buckets
+          - name: CEPH_BUCKET_PREFIX/ORG/REPO/ttm-model
```
Comment (oindrillac) [inline notebooks/time-to-merge-prediction/workshop/deployment-config.yaml:46]: maybe try

```suggestion
          - name: S3_MODEL_KEY
          - value: 'CEPH_BUCKET_PREFIX/ORG/REPO/ttm-model'
```
or
```suggestion
          - name: S3_MODEL_KEY
          - value: CEPH_BUCKET_PREFIX/ORG/REPO/ttm-model
```

**Review by oindrillac — COMMENTED** [review]
i think the format of the deployment config is wrong maybe thats why its failing

**Comment by suppathak** [comment]
> i think the format of the deployment config is wrong maybe thats why its failing

@oindrillac , True! I have just changed the format and tested by creating seldon deployment. Thank you!

**Review by oindrillac — APPROVED** [review]
/lgtm
/approve

**Pull request merged**